core/templates/domain/improvements/HighBounceRateTaskObjectFactory.ts

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Domain object for a high bounce-rate improvements task.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+import { ExplorationStats } from
+  'domain/statistics/ExplorationStatsObjectFactory';
+import { ITaskEntryBackendDict, TaskEntry } from
+  'domain/improvements/TaskEntryObjectFactory';
+import { ImprovementsConstants } from
+  'domain/improvements/improvements.constants';
+
+export class HighBounceRateTask extends TaskEntry {
+  constructor(
+      stateName: string,
+      issueDescription: string = null,
+      taskStatus: string = ImprovementsConstants.TASK_STATUS_TYPE_OBSOLETE,
+      closedBy: string = null,
+      closedOnMsecs: number = null) {
+    super(
+      ImprovementsConstants.TASK_TYPE_HIGH_BOUNCE_RATE,
+      ImprovementsConstants.TASK_TARGET_TYPE_STATE,
+      stateName, issueDescription, taskStatus, closedBy, closedOnMsecs);
+  }
+
+  public resolve(userId: string): void {
```

Comment:
Don't manipulate user IDs here. Call the backend (without a user ID) and let the backend handle the assignment. (The frontend can separately populate its own dict with the username.)

Reply:
Done.